html/index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>GWEALTH VISUALS CATALOGUE</title>
+</head>
+<body>
+	<header>
+		<h1>Gwealth Visuals</h1>
+		<nav>
+			<ul>
+				<li><a href="#">Portfolio</a></li>
+				<li><a href="#">Blog</a></li>
+				<li><a href="#">Brandings</a></li>
+				<li><a href="#">Showreel</a></li>
+				<li><a href="#">Testimonial</a></li>
+				<li><a href="#">Contact Us</a></li>
+			</ul>
+		</nav>
+	</header>
+	<section>
+		<article>
+			<h2>Branding</h2>
+			<p>Our branding service goes beyond just designing a logo. We help you develop a comprehensive brand strategy that encompasses all aspects of your business, from messaging to visual identity. We work to create a cohesive and memorable brand that resonates with your target audience and helps you stand out in a crowded marketplace.</p>
+		</article>
+		<article>
+			<h2>Printing and Publishing</h2>
+			<p>Our printing and publishing service offers high-quality printing services for a variety of materials, including business cards, brochures, flyers, and more. We also provide publishing services for books, magazines, and other print media. With our state-of-the-art equipment and experienced team, we can produce print materials that showcase your brand in the best possible light.</p>
+		</article>
+		<article>
+			<h2>Video Editing</h2>
+			<p>Our video editing service helps you create engaging and compelling video content that tells your brand's story. We work with you to develop a video concept, storyboard, and script, and then use our editing expertise to bring your vision to life. Whether you need a promotional video, product demo, or social media content, we can help you create video that resonates with your audience and drives results.</p>
+		</article>
+		<aside>
+			<p>Exciting news! GWealth Visuals now offers high-quality shirt printing with our new DTF machine. Contact us today to learn more</p>
+		</aside>
+		<ul>
+			<li>Quality Designs</li>
+			<li>Large Format Printing</li>
+			<li>Professional Corporate Video production</li>
+			<li><a href="https://www.facebook.com/gwealthvisuals/">Click to See More</a></li>
+		</ul>
+		<img src="../image/240.png" alt="my beautiful image">
+		<form>
+			<label for="name">Leave a comment</label>
+			<input type="text" id="name" name="name"><br><br>
+			<input type="submit" value="Submit">
+		</form>
+		<table>
+			<tr>
+				<th scope="col">Item Description</th>
+				<th scope="col">Price</th>
+				<th scope="col">Discounted price</th>
+			</tr>
+			<tr>
+				<td>All Branding</td>
+				<td>N50,000</td>
+				<td>N40,000</td>
+			</tr>
+			<tr>
+				<td>DTF A3 Size</td>
+				<td>N3,000</td>
+				<td>N2,500</td>
+			</tr>
+			<tr>
+				<td>Promo Videos</td>
+				<td>N10,000</td>
+				<td>N9,000</td>
+			</tr>
+		</table>
+	</section>
+	<footer>
+		<div>
```

Comment:
Why are you using different paragraphs for each single line text? You can use one paragraph and use <br/> to put line breaks if that's what you want to achieve